### PR TITLE
Add screencast docs

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #582

I had a chance to record some screencasts the other day.  I found the process enjoyable, so instead of just creating one getting started video I created a couple extra feature videos as well.

### Summary of Changes

* PR is blocked by actually uploading screencasts and linking to them
* Added a videos section
* Linked to Jason's overview
* Fixed some contributing TOC links
* Added a contributing sharing section

### Test Plan

- [x] test links
- [x] test videos

### CODE REVIEW

**Comment by jasonLaster** [comment]
@clarkbw mind reading this over. I feel like the language could use some cleaning up, especially the readme... :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,6 +48,15 @@ We are actively investigating ways of support enhancement requests in the projec
 
 Documentation is as important as code and we need your help to maintain clear and usable documentation.  If you find an error in here or other project documentation please [file an issue](https://github.com/devtools-html/debugger.html/issues/new) and tag it with the label [docs](https://github.com/devtools-html/debugger.html/labels/docs).
 
+### Share what you know
+
+You already know *far* more about the Debugger than the other people in your community.
+
```
Comment (clarkbw) [inline CONTRIBUTING.md]: Not sure about the language here.  I see what you're getting at but I'd reword it.

File: README.md [hunk 2]
```diff
@@ -28,6 +27,12 @@ cd debugger.html
 yarn start
 ```
 
+After the debugger is setup, you can:
```
Comment (clarkbw) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -53,6 +58,10 @@ We're all on Mozilla's IRC in the [#devtools-html][irc-devtools-html] channel on
 
 [getting-started]:./docs/getting-setup.md
 [contributing]:./CONTRIBUTING.md
+[getting-started-screencast]:/docs/videos#getting-started
+[up-for-grabs]:https://github.com/devtools-html/debugger.html/issues?label=???
```
Comment (clarkbw) [inline README.md]: Use this: https://github.com/devtools-html/debugger.html/issues?q=is%3Aissue+is%3Aopen+label%3A%22up+for+grabs%22

File: docs/getting-setup.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ Go to `localhost:8000` in any browser to view the Debugger. If everything worked
 
 ### Next Steps
 
-Go [here](./debugging-the-debugger.md) if you want to start debugging the debugger!
+Go [here][first-activity] if you want to start debugging the debugger!
```
Comment (clarkbw) [inline docs/getting-setup.md]: "Try this [first activity][first-activity] if you want..."?
